2022-10-24 15:49:30,043 INFO    MainThread:41576 [wandb_setup.py:_flush():71] setting env: {}
2022-10-24 15:49:30,043 INFO    MainThread:41576 [wandb_setup.py:_flush():71] setting login settings: {}
2022-10-24 15:49:30,043 INFO    MainThread:41576 [wandb_init.py:_log_setup():371] Logging user logs to C:\Users\albu\Documents\GitHub\deep-multimodal-glioma-prognosis\wandb\run-20221024_154930-34x0x343\logs\debug.log
2022-10-24 15:49:30,044 INFO    MainThread:41576 [wandb_init.py:_log_setup():372] Logging internal logs to C:\Users\albu\Documents\GitHub\deep-multimodal-glioma-prognosis\wandb\run-20221024_154930-34x0x343\logs\debug-internal.log
2022-10-24 15:49:30,044 INFO    MainThread:41576 [wandb_init.py:init():404] calling init triggers
2022-10-24 15:49:30,044 INFO    MainThread:41576 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-10-24 15:49:30,044 INFO    MainThread:41576 [wandb_init.py:init():424] re-initializing run, found existing run on stack: 2zb4cgft
2022-10-24 15:49:30,044 INFO    MainThread:41576 [wandb_run.py:finish():1457] finishing run tomealbuquerque/deep-multimodal-glioma-prognosis/2zb4cgft
2022-10-24 15:49:30,045 INFO    MainThread:41576 [jupyter.py:save_history():429] not saving jupyter history
2022-10-24 15:49:30,045 INFO    MainThread:41576 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-10-24 15:49:30,045 INFO    MainThread:41576 [wandb_init.py:_jupyter_teardown():303] cleaning up jupyter logic
2022-10-24 15:49:30,045 INFO    MainThread:41576 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-10-24 15:49:30,046 INFO    MainThread:41576 [wandb_run.py:_restore():1752] restore
2022-10-24 15:49:30,622 INFO    MainThread:41576 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 900
  total_bytes: 900
}

2022-10-24 15:49:30,734 INFO    MainThread:41576 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 900
  total_bytes: 900
}

2022-10-24 15:49:31,082 INFO    MainThread:41576 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 900
  total_bytes: 900
}

2022-10-24 15:49:31,967 INFO    MainThread:41576 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 900
  total_bytes: 8273
}

2022-10-24 15:49:32,077 INFO    MainThread:41576 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 900
  total_bytes: 8487
}

2022-10-24 15:49:32,186 INFO    MainThread:41576 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 900
  total_bytes: 8487
}

2022-10-24 15:49:32,294 INFO    MainThread:41576 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 900
  total_bytes: 8487
}

2022-10-24 15:49:32,403 INFO    MainThread:41576 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8487
  total_bytes: 8487
}

2022-10-24 15:49:32,511 INFO    MainThread:41576 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8487
  total_bytes: 8487
}

2022-10-24 15:49:32,620 INFO    MainThread:41576 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8487
  total_bytes: 8487
}

2022-10-24 15:49:32,728 INFO    MainThread:41576 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8487
  total_bytes: 8487
}

2022-10-24 15:49:32,837 INFO    MainThread:41576 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8487
  total_bytes: 8487
}

2022-10-24 15:49:33,204 INFO    MainThread:41576 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8487
  total_bytes: 8487
}

2022-10-24 15:49:33,782 INFO    MainThread:41576 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8487
  total_bytes: 8487
}
local_info {
}

2022-10-24 15:49:35,396 INFO    MainThread:41576 [wandb_run.py:_append_history():2130] rendering history
2022-10-24 15:49:35,396 INFO    MainThread:41576 [wandb_run.py:_append_summary():2085] rendering summary
2022-10-24 15:49:35,396 INFO    MainThread:41576 [wandb_run.py:_append_files():2180] logging synced files
2022-10-24 15:49:35,399 INFO    MainThread:41576 [wandb_init.py:init():460] starting backend
2022-10-24 15:49:35,400 INFO    MainThread:41576 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn, using: spawn
2022-10-24 15:49:35,404 INFO    MainThread:41576 [backend.py:ensure_launched():216] starting backend process...
2022-10-24 15:49:35,498 INFO    MainThread:41576 [backend.py:ensure_launched():221] started backend process with pid: 35724
2022-10-24 15:49:35,499 INFO    MainThread:41576 [wandb_init.py:init():469] backend started and connected
2022-10-24 15:49:35,500 INFO    MainThread:41576 [wandb_run.py:_label_probe_notebook():911] probe notebook
2022-10-24 15:49:35,500 INFO    MainThread:41576 [wandb_run.py:_label_probe_notebook():921] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-10-24 15:49:35,500 INFO    MainThread:41576 [wandb_init.py:init():533] updated telemetry
2022-10-24 15:49:35,559 INFO    MainThread:41576 [wandb_init.py:init():563] communicating current version
2022-10-24 15:49:37,617 INFO    MainThread:41576 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.13.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-10-24 15:49:37,618 INFO    MainThread:41576 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-10-24 15:49:37,998 INFO    MainThread:41576 [wandb_init.py:init():606] starting run threads in backend
2022-10-24 15:49:41,194 INFO    MainThread:41576 [wandb_run.py:_console_start():1810] atexit reg
2022-10-24 15:49:41,195 INFO    MainThread:41576 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-10-24 15:49:41,195 INFO    MainThread:41576 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-10-24 15:49:41,196 INFO    MainThread:41576 [wandb_run.py:_redirect():1745] Redirects installed.
2022-10-24 15:49:41,196 INFO    MainThread:41576 [wandb_init.py:init():633] run started, returning control to user process
2022-10-24 16:09:39,386 INFO    MainThread:41576 [wandb_setup.py:_flush():71] setting env: {}
2022-10-24 16:09:39,386 INFO    MainThread:41576 [wandb_setup.py:_flush():71] setting login settings: {}
2022-10-24 16:09:39,386 INFO    MainThread:41576 [wandb_init.py:_log_setup():371] Logging user logs to C:\Users\albu\Documents\GitHub\deep-multimodal-glioma-prognosis\wandb\run-20221024_160939-1l4oj1k0\logs\debug.log
2022-10-24 16:09:39,387 INFO    MainThread:41576 [wandb_init.py:_log_setup():372] Logging internal logs to C:\Users\albu\Documents\GitHub\deep-multimodal-glioma-prognosis\wandb\run-20221024_160939-1l4oj1k0\logs\debug-internal.log
2022-10-24 16:09:39,387 INFO    MainThread:41576 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000024A00435400>
2022-10-24 16:09:39,387 INFO    MainThread:41576 [wandb_init.py:init():404] calling init triggers
2022-10-24 16:09:39,387 INFO    MainThread:41576 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-10-24 16:09:39,387 INFO    MainThread:41576 [wandb_init.py:init():424] re-initializing run, found existing run on stack: 34x0x343
2022-10-24 16:09:39,387 INFO    MainThread:41576 [wandb_run.py:finish():1457] finishing run tomealbuquerque/deep-multimodal-glioma-prognosis/34x0x343
2022-10-24 16:09:39,388 INFO    MainThread:41576 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-10-24 16:09:39,388 INFO    MainThread:41576 [wandb_run.py:_restore():1752] restore
2022-10-24 16:09:40,166 INFO    MainThread:41576 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 900
  total_bytes: 900
}

2022-10-24 16:09:40,306 INFO    MainThread:41576 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 900
  total_bytes: 900
}

2022-10-24 16:09:40,593 INFO    MainThread:41576 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 900
  total_bytes: 900
}

2022-10-24 16:09:41,277 INFO    MainThread:41576 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 900
  total_bytes: 8247
}

2022-10-24 16:09:41,388 INFO    MainThread:41576 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 900
  total_bytes: 8463
}

2022-10-24 16:09:41,497 INFO    MainThread:41576 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 900
  total_bytes: 8463
}

2022-10-24 16:09:41,606 INFO    MainThread:41576 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 900
  total_bytes: 8463
}

2022-10-24 16:09:41,714 INFO    MainThread:41576 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8463
  total_bytes: 8463
}

2022-10-24 16:09:41,823 INFO    MainThread:41576 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8463
  total_bytes: 8463
}

2022-10-24 16:09:41,931 INFO    MainThread:41576 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8463
  total_bytes: 8463
}

2022-10-24 16:09:42,040 INFO    MainThread:41576 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8463
  total_bytes: 8463
}

2022-10-24 16:09:42,149 INFO    MainThread:41576 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8463
  total_bytes: 8463
}

2022-10-24 16:09:42,718 INFO    MainThread:41576 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8463
  total_bytes: 8463
}

2022-10-24 16:09:43,269 INFO    MainThread:41576 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8463
  total_bytes: 8463
}
local_info {
}

2022-10-24 16:09:44,973 INFO    MainThread:41576 [wandb_run.py:_append_history():2130] rendering history
2022-10-24 16:09:44,974 INFO    MainThread:41576 [wandb_run.py:_append_summary():2085] rendering summary
2022-10-24 16:09:44,974 INFO    MainThread:41576 [wandb_run.py:_append_files():2180] logging synced files
